multimodal/src/autogluon/multimodal/utils/inference.py

```diff
@@ -146,6 +147,22 @@ def infer_batch(
     return output[model.prefix]
 
 
+def run_model(model: nn.Module, batch: dict):
```

Comment:
Duplicate function? I seem this function also exists in `models/utils.py`.

Reply:
Nice catch! Yes, it is the one in `utils.py` that works. Import the funtion here would cause circular dependencies in fusion models.

Reply:
removed the duplicated function.